packages/wallet/frontend/src/components/settings/ChangePasswordForm.tsx

```diff
@@ -34,8 +19,20 @@ export const ChangePasswordForm = () => {
       </div>
       <Form
         form={form}
-        onSubmit={(data) => {
-          console.log(data)
+        onSubmit={async (data) => {
+          const response = await userService.changePassword(data)
+
+          if (response.success) {
+            openDialog(
+              <SuccessDialog
+                onClose={closeDialog}
+                title="Password updated sucessfully."
+                content="Please check your inbox. Click on the provided link and reset your password."
```

Comment:
There will be no email sent to change password, the form will appear in the Settings page.
As I mentioned there is a button - Change password, and then the form appears in the page with the 3 inputs.
And a new button of Save.

Reply:
Allright. I change it.